modules/mri_violations/test/mri_violationsTest.php

```diff
@@ -364,7 +364,7 @@ function testModuleLoadsWithAllSitesPermission()
         $bodyText = $this->safeFindElement(
             WebDriverBy::cssSelector("body")
         )->getText();
-        $this->assertNotContains(
+        $this->assertStringNotContainsString(
```

Comment:
Similarly, can we separate the phpunit upgrade from the php upgrade into different PRs?

Reply:
@driusan see https://github.com/aces/Loris/pull/7168 for the separate PR.